tests/eclair/Test/Eclair/LSP/JSONSpec.hs

```diff
@@ -46,7 +111,60 @@ spec = describe "LSP JSON processing" $ parallel $ do
       |]
 
   describe "JSON decoding" $ parallel $ do
-    it "can decode a command from JSON" pending
+    it "can decode a command from JSON" $ do
+      decodesAs
+        commandDecoder
+        [text|
+          {
+            "type": "hover",
+            "command": {
+              "position": {"line": 100, "column": 22},
+              "file": "/tmp/file.eclair"
+            }
+          }
+        |]
+        (Hover "/tmp/file.eclair" (SourcePos 100 22))
+
+    it "can decode a command from JSON" $ do
```

Comment:
Use unique names for the test names. Makes it easier to debug when 1 test fails.
For example: `can decode a document-highlight command from JSON` (also do this for other test names)

Reply:
Alternatively you can just group them all together under 1 `it` in this case (should not change that much anyway..)

Reply:
I modified the test names as suggested, thanks.